src/components/Tooltip/__tests__/Tooltip.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Tooltip from '../Tooltip';
+
+describe('Tooltip', () => {
+  it('should render as expected', () => {
+    const tree = renderer.create(<Tooltip></Tooltip>).toJSON();
```

Comment:
Quite a small component, but what do you think of adding a child to the component, so we see it wrapping it?

```suggestion
    const tree = renderer.create(<Tooltip>tooltip text</Tooltip>).toJSON();
```

Reply:
Yeah makes sense :) will add